update_version.yml

```diff
@@ -0,0 +1,29 @@
+name: Update Version on Docker Files
+
+env:
+  PRJ_VER: ${{ github.event.inputs.project_version }}
+
+
+
+on:
+ workflow_dispatch:
+    inputs:
+      project_version:
+        description: 'Version number to be released e.g 10.0.3'
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Install dependencies
+        run: sudo apt install libcurl4-openssl-dev libssl-dev
+
+      - name: Clone Tools branch
+        run: git clone --branch tools-transition-update-docker https://github.com/citusdata/tools.git
+        
+      - name: Install Python requirements
+        run: python -m pip install -r tools/python/requirements.txt
+        
+      - name: Update Docker Files
+        run: python -m python.update_docker --prj_ver ${PRJ_VER}
```

Comment:
I do not think this path is correct. I think you need `tools` in the path as well

Reply:
Also I am not comfortable with having directories called `python` and `bash` in our tools repository. The top directories used to be named after their purpose, not the language the files are written in. I think it makes more sense to move these two folders into `automated_packaging` folder there.

Reply:
automated packaging folder is copied under /usr/bin/ which is unnecessary for our case.
But I totally agree with you to change the directory name

Reply:
You cloned the tools repository but did not run `make install`. See https://github.com/citusdata/docker/pull/218/files#r613963456

Also let me make a clarification:

> automated packaging folder is copied under /usr/bin/ which is unnecessary for our case.

All the files in the automated_packaging except Makefile is copied. If you create some folders, they will not be copied for example. You will need to make some changes in the Makefiles to actually install python and bash scripts on separate directories.

See `man install` and https://github.com/citusdata/tools/blob/develop/automated_packaging/Makefile for more info

Reply:
@hanefi I suggest going on different directory other than automated_packaging until I complete the transition process. Conceptually seperating directiories makes my life easier when developing solutions. I renamed python to  package_automation